config/routes.rb

```diff
@@ -26,6 +26,16 @@
     delete 'logga_ut' => 'devise/sessions#destroy', as: :destroy_user_session
   end
 
+  # tags and exlinks
+  resources :ex_links
+
+  namespace :admin do
+    get 'ex_links/del_unused_tags' => 'ex_links#del_unused_tags'
+    get 'ex_links/check_dead' => 'ex_links#check_dead'
+    get 'ex_links/check_expired' => 'ex_links#check_expired'
+    resources :ex_links
```

Comment:
you can make this easier by writing:
```ruby
namespace :admin do
  resources :ex_links do
    get :del_unused_tags
    get :check_dead
    get :check_expired
  end
end
```
I think `del_unused_tags` should be a `delete`-action and the other two can be `patch`-actions.